app/precompiled.py

```diff
@@ -196,12 +196,10 @@ def sanitise_file_contents(encoded_string, *, allow_international_letters, filen
             "file": base64.b64encode(file_data.read()).decode('utf-8')
         }
     except Exception as error:
-        if isinstance(error, ValidationFailed):
```

Comment:
Nice one doing the work to reduce non-actionable errors. Wouldn't it be better though to add known errors to list of errors that only cause a warning, and if the error falls outside of those, still cause an exception? Then we would see if something unexpected happens.

Reply:
I thought about this some more and agree it's worth keeping the error-level log, but for a specific reason: if we suddenly have a large volume of unexpected exceptions, we will get notified via Elastalert.

I also thought more about `KeyError` and don't think it's best to keep erroring for that one: it probably is a bug with our code or the library we're calling, and ideally we should invest some time in trying to fix it.

I've rewritten the PR based on those thoughts.

What do you think?